app/src/main/res/values/strings.xml

```diff
@@ -991,6 +991,7 @@
     <string name="link_name">Link Name</string>
     <string name="delete_link">Delete Link</string>
     <string name="share_settings">Settings</string>
+    <string name="camera_permission_rationale">Please navigate to App info in settings and give permission manually.</string>
```

Comment:
Will be moved to localization PR if NC use different string value during/after merge same applicable for german strings.